types/adobe__pdfservices-node-sdk/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+        "no-unnecessary-class": false
```

Comment:
Instead of disabling the rule everywhere, please use `// tslint:disable-next-line:no-unnecessary-class` in the place(s) where it's needed.